test/integration/http2_integration_test.cc

```diff
@@ -195,6 +195,110 @@ TEST_P(Http2IntegrationTest, Trailers) { testTrailers(1024, 2048); }
 
 TEST_P(Http2IntegrationTest, TrailersGiantBody) { testTrailers(1024 * 1024, 1024 * 1024); }
 
+TEST_P(Http2IntegrationTest, IdleTimoutBasic) { testIdleTimeoutBasic(); }
```

Comment:
Please add a 1-2 sentence comment above each test saying what it tests. We're trying to do this for all new tests (and adding them to old tests whenever they are touched).